nixos/lib/default.nix

```diff
@@ -0,0 +1,46 @@
+let
+  # The warning is in a top-level let binding so it is only printed once.
+  inherit (import ../../lib) warn;
```

Comment:
You could `let` the import and use it in the arguments in line 6

Reply:
`import` is actually super cheap because Nix internally has a path to `Value` map. Expensiveness usually comes from a subsequent call, when the `Value` is a function, as is the case with `import <nixpkgs>` for example.

At first I figured naming lib here would be confusing, but it didn't occur to me that the name didn't have to be `lib`, which would have been shadowed.